rpcs3/Emu/Cell/Modules/cellAudio.cpp

```diff
@@ -846,16 +847,51 @@ void cell_audio_thread::operator()()
 		// Mix
 		float* buf = ringbuffer->get_current_buffer();
 
-		switch (cfg.audio_downmix)
+		switch (cfg.audio_channels)
```

Comment:
All of this was already done in get_channel_and_downmix.
I don't get why you're changing it

Reply:
What I mean is: why are you removing all the enums for numbers?
The enum tells us that there's 3 possible values.
Now it's just random numbers.
I'd like to keep both downmix and channels as enum